src/main/java/com/elytradev/smores/registries/FluidRegistry.java

```diff
@@ -27,9 +27,101 @@
 
 package com.elytradev.smores.registries;
 
+import com.elytradev.smores.Smores;
+import com.elytradev.smores.block.BlockFluidSmores;
+import com.elytradev.smores.materials.EnumAlloy;
+import com.elytradev.smores.materials.EnumMetal;
+import net.minecraft.util.ResourceLocation;
+import net.minecraftforge.fluids.Fluid;
+
+import java.util.ArrayList;
+import java.util.List;
+
 public class FluidRegistry {
 
+	private static final int[] MOLTEN_METAL_DENSITIES = {
```

Comment:
Densities and temperatures not tagged to any specific metal, so they're prone to breakage any time something gets added. These really belong as fields in their respective enum constant.

Reply:
Will do, didn't even think of that route.